### PR TITLE
Update B002 and make it more abstract.

### PR DESCRIPTION
**Pull request by littleali**
Make B002 more abstract so its title and description contains the goal of B002 not the way a team achieve this goal.

### CODE REVIEW

File: catalog/build_ci/B002.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ### B002
 
 |**Code**           | **B002** |
-|**Title**          | Automated Code Style Checking. |
-|**Description**    | |
+|**Title**          | Select and Enforce Code Style for Main Programming Languages. |
```
Comment (hamidrezash2000) [inline catalog/build_ci/B002.md:4]:
```suggestion
|**Title**          | Code Style Checking for Main Programming Languages. |
```
I prefer the item title not be a whole sentence as it was before.

Comment (emranbm) [reply catalog/build_ci/B002.md:4]: I disagree with the suggestion since the previous seems more clear and comprehensive, IMHO.

Comment (emranbm) [reply catalog/build_ci/B002.md:4]: As a compromise, I suggest this if you agree:
```suggestion
|**Title**          | Enforcing a Specific Code Style for Main Programming Languages. |
```

Comment (littleali) [reply catalog/build_ci/B002.md]: I replaced the title with @emranbm suggestion.

**Review by emranbm — APPROVED** [review]

**Review by emranbm — APPROVED** [review]

**Pull request merged**